scripts/machine_translation/dataprocessor.py

```diff
@@ -263,6 +263,6 @@ def write_sentences(sentences, file_path):
     with io.open(file_path, 'w', encoding='utf-8') as of:
         for sent in sentences:
             if isinstance(sent, (list, tuple)):
-                of.write(' '.join(sent) + '\n')
+                of.write(unicode(' '.join(sent) + '\n'))
```

Comment:
`unicode` is not defined in Py3. I don't think we need 'basic' strings here. If so, you may revert this change and add `from __future__ import unicode_literals` to the imports in the file

Reply:
I changed it in the patch